homeworks/01-bubble-sort/src/types/componentTypes.ts

```diff
@@ -0,0 +1,43 @@
+import { GameStatus } from "../helpers/enums";
+import { RandomSet } from "./serviceTypes";
+
+export type BubleSortProps = {}
```

Comment:
I would say that all types related to Props and State (I mean components types) should be located next to the appropriate component.

Inside the file or inside the component folder (eg, component `BubleSort` has its own folder with the same name `BubleSort` - `BubleSort/BubleSort.tsx` or `BubleSort/index.tsx`, and types for state/props should be there, eg, `BubleSort/types.ts` or `BubleSort/models.ts`)

Reply:
next time will definitely follow this kind of a folder structure